README.md

```diff
@@ -1,83 +1,100 @@
 # covidflow
 
-Flow for covid-19 assessment, navigation and follow-ups.
+Rasa backend for the covid-19 assessment, navigation and follow-ups available at https://covid19.dialogue.co.
 
-## Development
-
-### Make
-
-Make is expected to be installed on the developer workstation.
+## Folder Structure
 
-### Docker
+This repo is like a monorepo in that it contains multiple codebases. Here is an overview of the main folders content:
 
-Docker is expected to be installed on the developer workstation.
+- The `actions-server` and `core` folders contain the main components of covidflow, and they deploy together.
+- The `integration-tests-*` folders contain integration tests which are based on the Nu Echo's rasa-integration-testing framework (not open source yet, but available on pypi).
+- The `load-tests` folder contains a small integration of Locust with the rasa-integration-testing tools, allowing us to run load-tests using our test scenarios along with Locust.
+- **Apart from this project** but important to mention is the [covid-19](https://github.com/dialoguemd/covid-19) project, which contains the frontend that is connecting to covidflow.
 
-- _Windows_: Docker Desktop 2.1.0.5 is recommended
-- _Linux_: docker >= 19.03.5 and docker-compose >= 1.24.0
+A special note about the `actions-server` project: aside from serving as rasa actions-server, this project also contains:
 
-### Python version
+- `action-server/covidflow/jobs/send_reminders.py`: This script is used inside a helm-job for the purpose of sending daily assessment reminders by SMS. See `job.yml` for more details.
+- `action-server/alembic`: This is where the SQL migrations are located. See below for more details about the covidflow tables.
 
-Python version 3.6.8 is required to properly install Rasa components. It should be installed using [pyenv](https://github.com/pyenv/pyenv).
-
-### Environment initialization
+## Development
 
-When working in the project for the first time, the Python virtual environment (using `venv`) must be created by performing:
+### Prerequisites
 
-`make init`
+- Poetry - runing `make init` from either core or action-server project will install it.
+- Make is expected to be installed on the developer workstation.
```

Comment:
These sentences are redundant, maybe put the sentence "We expect the following ..." above the list